# req
String command
---
# res
bool result
